cmd/manager/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	"github.com/rigazilla/infinispan-operator/pkg/apis"
```

Comment:
This needs changing. Btw, is it possible to avoid using altogether github.com in the package names? This is likely to cause more problem in the future if we decide to fork or move...

Reply:
It looks like the go ecosystem does use heavily github.com in the package names, so let's follow  :)